apps/chat/src/components/Chat/ChatMessage/ChatMessage.tsx

```diff
@@ -133,69 +132,90 @@ export const ChatMessage: FC<Props> = memo(
+          <Menu
+            isTriggerEnabled={!isEditing}
+            placement="top-start"
+            listClassName="context-menu-chat bg-layer-3"
+            shouldFlip={false}
+            shouldApplySize={false}
+            style={{
+              top: `${clientY}px`,
+              left: `${clientX}px`,
+            }}
+            type="contextMenu"
+            className="w-full text-start"
+            enableAncestorScroll
+            noFocusReturn
+            trigger={
+              <ChatMessageContent
+                isLastMessage={isLastMessage}
+                messageIndex={messageIndex}
+                conversation={conversation}
+                isEditing={isEditing}
+                toggleEditing={toggleEditing}
+                toggleEditingTemplates={toggleEditingTemplates}
+                isEditingTemplates={isTemplateModalOpened}
+                message={message}
+                onEdit={onEdit}
+                onClick={(e, messageRef) => {
+                  const rect = messageRef.current!.getBoundingClientRect();
+                  setClientY(e.clientY - rect.y);
+                  setClientX(
+                    e.clientX -
+                      rect.x -
+                      (e.pageX > window.innerWidth / 2
+                        ? onRegenerate
+                          ? CONTEXT_MENU_REGENERATE_OFFSET
+                          : CONTEXT_MENU_OFFSET
+                        : 0),
+                  );
+                }}
+                {...props}
+              />
+            }
+          >
+            <MessageMobileButtons
+              isMessageStreaming={!!conversation.isMessageStreaming}
               isLastMessage={isLastMessage}
-              messageIndex={messageIndex}
-              conversation={conversation}
+              message={message}
+              onCopy={handleCopy}
+              messageCopied={messageCopied}
+              editDisabled={editDisabled}
+              onLike={onLike}
+              onDelete={() => setIsDeleteConfirmationOpened(true)}
               isEditing={isEditing}
               toggleEditing={toggleEditing}
-              message={message}
-              onEdit={onEdit}
-              onClick={(e, messageRef) => {
-                const rect = messageRef.current!.getBoundingClientRect();
-                setClientY(e.clientY - rect.y);
-                setClientX(
-                  e.clientX -
-                    rect.x -
-                    (e.pageX > window.innerWidth / 2
-                      ? onRegenerate
-                        ? CONTEXT_MENU_REGENERATE_OFFSET
-                        : CONTEXT_MENU_OFFSET
-                      : 0),
-                );
-              }}
-              {...props}
+              onRegenerate={onRegenerate}
+              isConversationInvalid={isConversationInvalid}
+              isEditTemplatesAvailable={isMessageTemplatesEnabled}
+              toggleTemplatesEditing={toggleEditingTemplates}
```

Comment:
```suggestion
              onToggleTemplatesEditing={toggleEditingTemplates}
```

lets add `on` to follow convention